Makefile

```diff
@@ -60,5 +60,5 @@ k8s: validate
 	$(MAKE) k8s kubernetes_version=1.18.9 kubernetes_build_date=2020-11-02 pull_cni_from_github=true
 
 .PHONY: 1.19
-	1.19:
-	    $(MAKE) k8s kubernetes_version=1.19.6 kubernetes_build_date=2021-01-05 pull_cni_from_github=true
+1.19:
```

Comment:
Can you also update `make all` on line :32 to add 1.19?

Reply:
@heybronson done